src/dftbp/dftbplus/parser.F90

```diff
@@ -1314,6 +1318,8 @@ subroutine readDFTBHam(node, ctrl, geo, slako, poisson)
           end if
           strTmp = trim(prefix) // trim(elem1) // trim(separator) &
               &// trim(elem2) // trim(suffix)
+          call findFile(searchPath, strTmp, strOut)
+          if (allocated(strOut)) strTmp = strOut
```

Comment:
```suggestion
          if (allocated(strOut)) call move_alloc(strOut, strTmp)
```

Reply:
Doesn't work because strTmp is not allocatable.